### PR TITLE
Fixed Environment Scale

### PR DESCRIPTION
**Pull request by bestmazzo**

### CODE REVIEW

**Comment by amenak77** [comment]
Thanks, Matteo; you found a little time, after all...  :)

File: src/utils/fd-axios.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ let FD_API_URL
 if (window.location.host === 'fd-vue-webapp.herokuapp.com') {
   FD_API_URL = 'http://174.138.4.3:9111/v3'
 } else {
-  FD_API_URL = 'http://' + window.location.hostname + ':9111/v3'
+  FD_API_URL = 'http://174.138.4.3:9111/v3'
```
Comment (P3trur0) [inline src/utils/fd-axios.js]: Is this change here actually relevant? I think this shall introduce a wrong behavior.

Comment (mcicolella) [reply src/utils/fd-axios.js]: Fixed in 2cb69782bd6cb703fe3a44e0d250aac56db8fa48

Comment (P3trur0) [reply src/utils/fd-axios.js]: :+1:

**Pull request merged**